app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render partial: 'devise/shared/header', locals: { heading: 'Log In'}%>
```

Comment:
- [ ] Kindly follow the same layout as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Layout%3A%20composition%20and%20space%20between%20elements.) of the page for the login page.

| Your design | Design Guidelines |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/210605745-4403230d-8210-4580-a6ee-f4e65853290a.png) | ![image](https://user-images.githubusercontent.com/24703171/210605595-55861682-29b0-4dd8-8faa-be3cbfdc88de.png) |

Reply:
Thank you for the review. Updated PR with requested changes :)